Portfolio.css

```diff
@@ -1,3 +1,173 @@
+* {
+    box-sizing: border-box;
```

Comment:
Might want to consider adding vendor prefixes to this

Reply:
Normalise should already fix this, I think.